lib/lita/handlers/stats.rb

```diff
@@ -0,0 +1,49 @@
+require 'signalfx'
+
+class Stats
+
+  @@signalfx_client ||= SignalFx.new ENV['LITA_SIGNALFX_TOKEN']
```

Comment:
IMO I think it is a little more idiomatic to wrap this in an accessor method.
```
def self.signalfx_client
  @signalfx_client ||= SignalFx.new ENV['LITA_SIGNALFX_TOKEN']
end
```

Then in other methods you can call `signalfx_client.send(...)`.

Reply:
do you mean something like that?
https://github.com/change/lita-totems/pull/2/files/fe5dd8636b5d5c4f9a67237558b736e59494750c#diff-868c63eb562ba03146634af0ff2cbf9e892e7fd022098d3ae88dd01511befe94

Reply:
Like that, but as a class method instead of an instance method.